Gemfile

```diff
@@ -9,30 +9,30 @@ gem 'pg', '~> 1.1'
 
 gem 'puma', '~> 5.0'
 
-group :development, :test do
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
-end
-
 group :development do
   gem 'listen', '~> 3.3'
 end
 
 group :test do
-  # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '>= 3.26'
-  gem 'selenium-webdriver'
-  # Easy installation and use of web drivers to run system tests with browsers
-  gem 'webdrivers'
+  gem 'json_matchers'
+  gem 'shoulda-matchers', '~> 4.0'
 end
 
 group :development, :test do
   gem 'brakeman'
   gem 'bundler-audit', '~> 0.8.0'
+  gem 'faker', git: 'https://github.com/faker-ruby/faker.git'
```

Comment:
Just `gem 'faker', '~> 2.18'`. Also is very good practice to specify versions for each gem into `Gemfile` manifest.

Reply:
I'll add it, but for faker there is no 2.18 version in ruby gems, because of that we use this git link